pitop/robotics/json/alex.json

```diff
@@ -0,0 +1,43 @@
+{
+    "classname": "Pitop",
```

Comment:
Configurations should have a schema version associated with them, in case of breaking changes in the future.